src/libs/OptionsListUtils.js

```diff
@@ -307,6 +307,25 @@ function isCurrentUser(userDetails) {
     return result;
 }
 
+/**
+ * Returns whether the given userDetails is excluded from IOU request
+ * @param {Objecy} userDetails
+ *  @returns {Bool}
+ */
+function isExcludedIOUUsers(userDetails) {
+    if (isCurrentUser(userDetails)) {
+        // Currently logged in user is excluded
+        return true;
+    }
+
+    if (userDetails && userDetails.login) {
+        const login = userDetails.login.toLowerCase();
+        return login === CONST.EMAIL.CHRONOS || login === CONST.EMAIL.CONCIERGE || login === CONST.EMAIL.RECEIPTS;
```

Comment:
Something like this is a bit more readable to me. And we could also make the array a constant like `EXPENSIFY_EMAILS`.

```js
_.contains([CONST.EMAIL.CHRONOS, CONST.EMAIL.CONCIERGE, CONST.EMAIL.RECEIPTS], login);
```

Reply:
For adding `EXPENSIFY_EMAILS` in `CONST`,
with underscore (we need to import underscore)
```
get EXPENSIFY_EMAILS() {
        return _.map(_.allKeys(this.EMAIL), key => this.EMAIL[key]);
},
```
Without _ underscore library
```
get EXPENSIFY_EMAILS() {
        return Object.keys(this.EMAIL).map(key => this.EMAIL[key]);
},
```
which way do you prefer @marcaaron ?

Reply:
`_.keys()` is what we use instead of `Object.keys()`

Reply:
but `_.map()` also works on object values

Reply:
I'm not sure I'd do what you are suggesting though because if someone adds a new `CONST.EMAIL.SOMETHING` it will automatically get excluded from the IOUs (which might not be intended). Better to be specific I think.

Reply:
So with the underscore library then, I'll update a PR Soon!

Reply:
Yeah, but I'm also saying we can skip underscore entirely and just do something explicit like this

```js
get EXPENSIFY_EMAILS() {
    return [
        this.EMAIL.CHRONOS, 
        this.EMAIL.CONCIERGE, 
        this.EMAIL.RECEIPTS,
    ];
}
```

Reply:
> I'm not sure I'd do what you are suggesting though because if someone adds a new CONST.EMAIL.SOMETHING will automatically get excluded from the IOUs (which might not be intended). Better to be specific I think.

Yes, it might be unintended, so like you said we can keep it specific. Then this should work.
```
get EXPENSIFY_EMAILS() {
        return [
            this.EMAIL.CHRONOS,
            this.EMAIL.CONCIERGE,
            this.EMAIL.RECEIPTS,
        ];
    },
```